docker/README.md

```diff
@@ -29,18 +29,18 @@ The base flavor contains PyTorch pre-installed; the vision flavor also includes
 To install composer, once inside the image, run `pip install mosaicml`.
 
 <!-- BEGIN_PYTORCH_BUILD_MATRIX -->
-| Linux Distro   | Flavor   | PyTorch Version   | CUDA Version        | Python Version   | Docker Tags                                                                                       |
-|----------------|----------|-------------------|---------------------|------------------|---------------------------------------------------------------------------------------------------|
-| Ubuntu 20.04   | Base     | 2.0.1             | 11.8.0 (Infiniband) | 3.10             | `mosaicml/pytorch:2.0.1_cu118-python3.10-ubuntu20.04`                                             |
-| Ubuntu 20.04   | Base     | 2.0.1             | 11.8.0 (EFA)        | 3.10             | `mosaicml/pytorch:2.0.1_cu118-python3.10-ubuntu20.04-aws`                                         |
-| Ubuntu 20.04   | Base     | 2.0.1             | cpu                 | 3.10             | `mosaicml/pytorch:2.0.1_cpu-python3.10-ubuntu20.04`                                               |
-| Ubuntu 20.04   | Base     | 1.13.1            | 11.7.1 (Infiniband) | 3.10             | `mosaicml/pytorch:latest`, `mosaicml/pytorch:1.13.1_cu117-python3.10-ubuntu20.04`                 |
-| Ubuntu 20.04   | Base     | 1.13.1            | 11.7.1 (EFA)        | 3.10             | `mosaicml/pytorch:latest-aws`, `mosaicml/pytorch:1.13.1_cu117-python3.10-ubuntu20.04-aws`         |
-| Ubuntu 20.04   | Base     | 1.13.1            | cpu                 | 3.10             | `mosaicml/pytorch:latest_cpu`, `mosaicml/pytorch:1.13.1_cpu-python3.10-ubuntu20.04`               |
-| Ubuntu 20.04   | Vision   | 2.0.1             | 11.8.0 (Infiniband) | 3.10             | `mosaicml/pytorch_vision:2.0.1_cu118-python3.10-ubuntu20.04`                                      |
-| Ubuntu 20.04   | Vision   | 2.0.1             | cpu                 | 3.10             | `mosaicml/pytorch_vision:2.0.1_cpu-python3.10-ubuntu20.04`                                        |
-| Ubuntu 20.04   | Vision   | 1.13.1            | 11.7.1 (Infiniband) | 3.10             | `mosaicml/pytorch_vision:latest`, `mosaicml/pytorch_vision:1.13.1_cu117-python3.10-ubuntu20.04`   |
-| Ubuntu 20.04   | Vision   | 1.13.1            | cpu                 | 3.10             | `mosaicml/pytorch_vision:latest_cpu`, `mosaicml/pytorch_vision:1.13.1_cpu-python3.10-ubuntu20.04` |
+| Linux Distro   | Flavor   | PyTorch Version   | CUDA Version        | Python Version   | Docker Tags                                                                                      |
+|----------------|----------|-------------------|---------------------|------------------|--------------------------------------------------------------------------------------------------|
+| Ubuntu 20.04   | Base     | 2.0.1             | 11.8.0 (Infiniband) | 3.10             | `mosaicml/pytorch:latest`, `mosaicml/pytorch:2.0.1_cu118-python3.10-ubuntu20.04`                 |
```

Comment:
tbh i don't think we should move latest to torch2 until we figure out the checkpoint memory issue.